src/utils/response/apiResponse.ts

```diff
@@ -21,4 +121,16 @@ export type ApiResponseOnFailure = {
   ErrorInternal?: string;
 };
 
-export type ApiResponse<T> = ApiResponseOnSuccess<T> | ApiResponseOnFailure;
+export type BaseApiResponse<ResourceName extends ResourceNames> = {
+  ResourceName: ResourceName;
+  RequestStartTime: string;
+};
+
+export type ApiResponse<ResourceName extends ResourceNames> = BaseApiResponse<ResourceName> &
+  (ApiResponseOnSuccess<ResourceName> | ApiResponseOnFailure);
+
+export type LibraryReturn<ResourceName extends ResourceNames, Results = Resources[ResourceName]> = {
+  Results: Results;
+  Deleted: ListOfAllResources;
```

Comment:
same here about Deleted

Reply:
LibraryReturn type is return type of js-client-library.
Is always there because of convenience, of not having to double checking. Was discussed during frontend meeting, we can have a quick chat on the topic again, maybe we had some missunderstanding.